src/protocols/ab/eip_cip.c

```diff
@@ -996,6 +999,7 @@ int build_write_request_connected(ab_tag_p tag, int byte_offset)
     ab_request_p req = NULL;
     int multiple_requests = 0;
     int write_size = 0;
+    int pad_to_even_bytes = 1;
```

Comment:
Is there a reason you were inconsistent with the naming?   I actually like this better than the 16 name...  Just wondering if I am missing something here.

Reply:
Nope, that was just a mistake on my behalf. I was planning to go ahead with your idea of changing the parameter to str_pad_to_multiple_bytes anyway, as that seems like an improvement and will be quite simple to implement.